pkgs/by-name/na/nautilus-open-in-blackbox/package.nix

```diff
@@ -0,0 +1,33 @@
+{ python3, fetchFromGitHub, gnome, stdenv, lib }:
+stdenv.mkDerivation rec {
+  pname = "nautilus-open-in-blackbox";
+  version = "0.1.1";
+
+  src = fetchFromGitHub
+    {
+      owner = "ppvan";
+      repo = pname;
+      rev = version;
+      hash = "sha256-5rvh3qNalpjamcBVQrnAW6GxhwPPlRxP5h045YDqvrM=";
+    };
```

Comment:
```suggestion
  src = fetchFromGitHub {
    owner = "ppvan";
    repo = pname;
    rev = "refs/tags/${version}";
    hash = "sha256-5rvh3qNalpjamcBVQrnAW6GxhwPPlRxP5h045YDqvrM=";
  };
```

Reply:
Part of this suggestion was moving the opening brace in-line with fetchFromGitHub and adjusting indentation accordingly.. While we don't have official formatting yet, and I'm not a fan of nit-picking formatting, I do think it's important to understand this is the de facto standard in nixpkgs.

Reply:
Sorry, I looked over the formatting, I think It was auto formatted. Anyways I have fixed the formatting